anylink/forms.py

```diff
@@ -0,0 +1,41 @@
+from django import forms
+from django.conf import settings
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _
+
+from .compat import get_all_related_objects, add_error
+from .models import AnyLink
+
+
+class AnyLinkAdminForm(forms.ModelForm):
+
+    confirmation = forms.BooleanField(widget=forms.HiddenInput(), required=False)
+
+    class Meta:
+        model = AnyLink
+
+    def clean(self):
+        data = self.cleaned_data
+
+        if self.instance.pk:
+            if settings.ANYLINK_REUSABLE:
```

Comment:
Setting in ANYLINK_ALLOW_MULTIPLE_USE umbenennen. Vergleich vereinfachen "if self.instance.pk and setting.ANYLINK_ALLOW_MULTIPLE_USE"